CSharp/Samples/Microsoft.Bot.Connector.Teams.SampleBot/Web.config

```diff
@@ -20,7 +24,7 @@
   -->
   <system.web>
     <compilation debug="true" targetFramework="4.6.1" />
-    <httpRuntime targetFramework="4.6" />
+    <httpRuntime targetFramework="4.6" requestPathInvalidCharacters=""/>
```

Comment:
Let's just exclude the characters we need and add everything else back here. Also put a comment describing why we need this.